code/modules/mob/living/living_say.dm

```diff
@@ -156,9 +156,13 @@ GLOBAL_LIST_INIT(message_modes_stat_limits, list(
 		language = get_selected_language()
 	var/mob/living/carbon/human/H = src
 	if(!can_speak_vocal(message))
-		if (HAS_TRAIT(src, TRAIT_SIGN_LANG) && H.mind.miming)
-			to_chat(src, "<span class='warning'>You stop yourself from signing in favor of the artform of mimery!</span>")
-			return
+		if(H.mind?.miming)
+			if(HAS_TRAIT(src, TRAIT_SIGN_LANG))
+				to_chat(src, "<span class='warning'>You stop yourself from signing in favor of the artform of mimery!</span>")
+				return
+			else
+				to_chat(src, "<span class='green'>Your vow of silence prevents you from speaking!</span>")
```

Comment:
Why green when the others are `warning`?

Reply:
I made it green because the vow is a good thing for mimes so it should be a positive to_chat.

Reply:
I'm not super committed to it, I just thought that it made a kind of sense to have "You were just saved from speaking by your vow" as a positive thing seeing as how important and unique the vow is.

If you want it back to warning I'll set it as warning, no big.

Reply:
sorry i cant read